ods_ci/tasks/Resources/Provisioning/GPU/NFD/install_nfd.sh

```diff
@@ -5,7 +5,7 @@ NFD_INSTALL_DIR="$(dirname "$0")"
 NFD_INSTANCE=$NFD_INSTALL_DIR/nfd_deploy.yaml
 echo "Installing NFD operator"
 oc apply -f "$NFD_INSTALL_DIR/nfd_operator.yaml"
-oc wait --timeout=3m --for jsonpath='{.status.state}'=AtLatestKnown -n openshift-nfd sub nfd
+oc wait --timeout=20m --for jsonpath='{.status.state}'=AtLatestKnown -n openshift-nfd sub nfd
```

Comment:
20minutes sound a lot, did you wait for 20m for NFD to be ready?

Reply:
I don't recall the exact time but this will give us less chances of failing in the happy case